src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -245,13 +227,13 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None):
             if stride_length_s is None:
                 stride_length_s = chunk_length_s / 6
 
-            chunk_len = int(round(chunk_length_s * self.feature_extractor.sampling_rate))
-
             if isinstance(stride_length_s, (int, float)):
                 stride_length_s = [stride_length_s, stride_length_s]
 
-            stride_left = int(round(stride_length_s[0] * self.feature_extractor.sampling_rate))
-            stride_right = int(round(stride_length_s[1] * self.feature_extractor.sampling_rate))
+            align_to = self.model.config.inputs_to_logits_ratio if self.type != "seq2seq" else 1
```

Comment:
(nit) -  does `align_to=1` make sense here for Seq2Seq? I think the whole if statement can't currently be reached when model is Seq2Seq no? Maybe better to raise here instead?

Reply:
I think you're correct, I moved to adding simply a comment so future readers know that `seq2seq` is supposed to be taken care of before.